modules/apps/portal-search/portal-search/src/main/java/com/liferay/portal/search/internal/custom/relevance/CustomRelevanceContainer.java

```diff
@@ -45,6 +46,22 @@ public void addBoosterTerms(BooleanQuery booleanQuery) {
 		}
 	}
 
+	@SuppressWarnings("unchecked")
+	public void addCustomBoosterTerms(
+		ArrayList<Object[]> customBoostList, BooleanQuery booleanQuery) {
+
+		if (customBoostList != null) {
+			for (Object[] customBoost : customBoostList) {
+				String field = (String)customBoost[0];
```

Comment:
@brandizzi an we move the class "CustomRelevance" to the "portal-search-api" module to use it here?

Reply:
@luanmaoski yes, that makes sense!